common/static/common/js/spec/components/paging_spec.js

```diff
@@ -73,19 +73,19 @@ define([ "jquery", "common/js/spec_helpers/ajax_helpers", "URI",
                 describe("setPage", function () {
                     it('can set the current page', function () {
                         var requests = AjaxHelpers.requests(this);
-                        pagingView.setPage(0);
+                        pagingView.setPage(1);
```

Comment:
We changed the interface so that setPage always exposes one indexed page numbers, regardless of the underlying server API. These tests were written by directly checking currentPage on the collection, which is zero indexed in this case, so the numbers appear different. The way the new tests do this is by checking currentOneIndexPage() instead, but I wanted to change these tests as little as possible.